wp-content/themes/haruoshi_origin/src/scss/style.scss

```
@@ -1,3 +1,30 @@
+:root {
+    --header-height: 10%;
+    --footer-height: 3%;
+  }
```

ヘッダー、フッターの高さは％じゃないほうがいいと思うよ
そんで、scssなら変数宣言できるから
```suggestion
$header-height: 40rem;
$footer-height: 20rem;
```
みたいな感じ。